### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by jv1968** [yinu0x21/MockLocation#1]
Hi,
Thank you for all your hard work!
Is it possible to give some instructions on how to properly install this?
Thanks

**Comment by iqfareez** [comment]
Yeah. I've been wondering too seems I can't find any apk lying around this repo
